### PR TITLE
Add "Get started with GPU Compute on the Web" article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds the "Get started with GPU Compute on the Web" article.

**Target Live Date:** End of august.

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

Live preview: https://pr-8010-dot-web-central.appspot.com/web/updates/2019/08/get-started-with-gpu-compute-on-the-web

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@jpmedley There are still some TODOs left to address but I think we're in good shape now. WDYT?

File: src/content/en/updates/2019/08/get-started-with-gpu-compute-on-the-web.md [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+      alt="Matrix multiplication diagram">
+  <figcaption>
+    <b>Figure 1.</b>
+    Matrix multiplication diagram
+  </figcaption>
+</figure>
+
+In short, here’s what we’re going to do: 
+
+- Create three GPU buffers (two for the matrices to multiply and one for the
```
Comment (jpmedley) [inline src/content/en/updates/2019/08/get-started-with-gpu-compute-on-the-web.md]: If this is a procedure, these need to be numbers not bullets.

**Comment by jpmedley** [comment]
@beaufortfrancois Note that there is a [PR against this](https://github.com/google/WebFundamentals/pull/8011) with some last minute edits.

**Comment by jpmedley** [comment]
Otherwise, it looks good.

**Pull request merged**